modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -99,32 +109,42 @@ Example DataWeave 2.0 expression to be used when configuring the policy:
 #[payload.client_secret]
 ----
 
-== Credentials as Basic Authorization header
+As shown in the above example, it is also possible to configure the policy to obtain the credentials from the request payload. MuleSoft does not recommend this option because it is harder to reflect in the API specification.
+
+==== With Basic Authorization header
+
 Example request using curl:
 [source,text]
 ----
 curl "http://localhost/myResource" -u 1234:abcd
 ----
 
-Example DataWeave 2.0 expression to be used when configuring the policy:
+=== Configure API Specifications
+
+The Client ID Enforcement policy requires changes in your API specification to implement the credentials requirement.
+You can find the RAML or OAS snippet link containing the RAML or OAS code you need to add to the API specification in the corresponding policy. You can access this code from the list of applied policies in the Policies tab of your API specification in API Manager.
+//future enhancement: how to update API spec once a policy is applied.
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+=== Policy Output
 
-== WWW Authenticate HTTP header
-When a HTTP request is performed against a protected resource, where the Client ID enforcement policy is applied, and client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
+When an HTTP request is performed against a protected resource for which the Client ID Enforcement policy is applied, and the client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
 
 [width="100%", cols="2,70"]
 |===
 >s| Custom mode | Header 'WWW-Authenticate'='Client-ID-Enforcement'
 >s| Basic Auth mode | Header 'WWW-Authenticate'='Basic realm="mule-realm"
 |===
 
```

Comment:
"custom mode" and "basic auth mode" do not exactly match how these two are referred to earlier.
Avoid using more than one term for one thing, GLOBALLY. It confuses readers.

Reply:
I'm checking with Eva! Thanks!